crates/sui-bridge/src/events.rs

```diff
@@ -83,7 +83,10 @@ impl TryFrom<MoveTokenBridgeEvent> for EmittedSuiToEthTokenBridgeV1 {
         })?;
 
         match sui_chain_id {
-            BridgeChainId::SuiMainnet | BridgeChainId::SuiTestnet | BridgeChainId::SuiDevnet => {}
+            BridgeChainId::SuiMainnet
```

Comment:
I don't have the full picture of the logic, but If Sui and Eth chain IDs are in separate enums, these invariants would be enforced statically. Also I wonder if it would be less error prone to hard code the pairs of chain IDs that make sense, e.g. (SuiMainnet, EthMainnet), (SuiTestnet, EthSepolia), (SuiLocalTest, EthLocalTest).

Reply:
with this the trade-off is we need multiple enums when adding new chains in the future, and need to make sure their value do not collide.

I think a bette approach may be to add a `fn chain_type() -> ChainType` for `BridgeChainId` enum. I haven't felt too bad about this yet though